chatmd/importers/utils.py

```
@@ -0,0 +1,97 @@
+    per_chunk_links.setdefault(chunk_index, []).append((attachment_name, rel))
+    head = lines[:PREVIEW_LINES]
+    tail = lines[-PREVIEW_LINES:]
+    preview = "\n".join(
+        head
+        + [
+            "…",
+            "",
+            f"(Full content saved to {attachment_name})",
+            "",
+        ]
+        + tail
```

Ruff (0.15.9)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)